spec/core/middleware/response/sanitise_response_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'core/spec_helper'
+
+describe ZendeskAPI::Middleware::Response::SanitizeResponse do
+  def fake_response(data)
+    stub_json_request(:get, %r{blergh}, data)
+    response = client.connection.get('blergh')
+    expect(response.status).to eq(200)
+    response
+  end
+
+  describe 'with bad characters' do
+    let(:response) { fake_response("{\"x\":\"2012-02-01T13:14:15Z\", \"y\":\"awesome!\ud83d\udc4d\"}") }
```

Comment:
throwing some actual utf-8 in the test might reveal some bad behavior `ÍÎ˛¸◊Ç˛◊´„`